tiatoolbox/annotation/storage.py

```diff
@@ -2610,8 +2696,8 @@ def _kind_of_pquery(
 
     @staticmethod
     def _validate_select_where_type(
-        select: Union[str, bytes, Callable],
-        where: Union[str, bytes, Callable],
+        select: str | bytes | Callable,
+        where: str | bytes | Callable,
```

Comment:
Select and Predicate custom types

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/991322d859c0c22e7c7a1d713362944aa4317198